README.md

```diff
@@ -13,6 +13,7 @@ It enables users to write pure Python code to project graphs, run algorithms, as
 
 The API is designed to mimic the GDS Cypher procedure API in Python code.
 It abstracts the necessary operations of the [Neo4j Python driver](https://neo4j.com/docs/python-manual/current/) to offer a simpler surface.
+Additionally, the client-specific graph, model, and pipeline objects offer convenient functions that heavily reduce the need to use Cypher to access and operate these GDS resources.
```

Comment:
Nice!

Reply:
I wonder if we should also add this addition to the front page of the client manual?

Reply:
Sure, I can do that.